shell/artifacts/Social/Post.schema

```diff
@@ -20,7 +20,10 @@ schema Post
   // polymorphic display of and interaction with the post, as the
   // particles creating or updating the Post entity can set whatever
   // recipe they deem appropriate for the Post (or Post subclass).
-  Text recipe
+  Text renderRecipe
+  // A JSON representation of the primary particle referenced in
+  // |renderRecipe| intended for use in displaying this post.
+  Text renderParticleSpec
```

Comment:
isn't writing the particle spec as part of the `renderRecipe` enough?

Reply:
We need (or, at least, I thought we needed) the `ParticleSpec` in `MultiplexerDomParticle` to build `otherMappedViews` and `otherConnections` on a per-item/render-particle basis.

This need may go away with the work noted in https://github.com/PolymerLabs/arcs/pull/1112 re: `onArcLoadRecipe`, will know more soon.

Reply:
That is, the particle spec written in `renderRecipe` is the particle definition, but that's not the same as having a `ParticleSpec` instance whose connections and types can be scrutinized, and `ParticleSpec.toLiteral` and `fromLiteral` was the recommended way to accomplish.